safenode/src/transfers/mod.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+mod error;
+
+pub(crate) use self::error::{Error, Result};
+
+use crate::{
+    protocol::types::{
+        address::DbcAddress,
+        fees::{FeeCiphers, RequiredFee, SpendPriority, SpendQ},
+    },
+    storage::SpendStorage,
+};
+
+use sn_dbc::{DbcId, DbcTransaction, MainKey, SignedSpend, Token};
+
+use std::collections::{BTreeMap, BTreeSet};
+use xor_name::XorName;
+
+const STARTING_FEE: u64 = 4000; // 0.000004 SNT
```

Comment:
These kind of hardcoded values should probably be in a config file right?

Reply:
This is obviously temporary. This value is completely out of the blue.